### PR TITLE
Add project proposal for Thanos

### PR DESCRIPTION
**Pull request by fpetkovski**
cc @saswatamcode @bwplotka

### CODE REVIEW

**Review by bwplotka — APPROVED** [review]
Nice! Thx!

File: lfx-mentorship/2022/03-Sept-Nov/project_ideas.md [hunk 1]
```diff
@@ -187,4 +187,13 @@ a new and popular computing platform, Volcano should integrate with it.
 - Description: Users have no ways to update log level of Volcano components now, which is difficult to track bugs especially in the production environment. Your
 task is to achieve it.
 - Mentor(s): [william-wang](https://github.com/william-wang)
-- Issue: <https://github.com/volcano-sh/volcano/issues/2430>
\ No newline at end of file
+- Issue: <https://github.com/volcano-sh/volcano/issues/2430>
+
+#### Thanos
+
+##### Support for tenant-specific external labels in Thanos Receiver
```
Comment (nate-double-u) [inline lfx-mentorship/2022/03-Sept-Nov/project_ideas.md:194]: Is this a placeholder title?

**Review by nate-double-u — COMMENTED** [review]
Hi @fpetkovski, thanks for this! I've got one question about your PR:

File: lfx-mentorship/2022/03-Sept-Nov/project_ideas.md (hunk 1 above)
Comment (fpetkovski) [reply lfx-mentorship/2022/03-Sept-Nov/project_ideas.md]: Ah sorry, it was a duplicated title. I've removed one of them :)

**Pull request merged**